plugin-server/src/worker/ingestion/event-pipeline/determineShouldBufferStep.ts

```diff
@@ -0,0 +1,52 @@
+import { DateTime } from 'luxon'
+
+import { Hub, Person, PreIngestionEvent, TeamId } from '../../../types'
+import { EventPipelineRunner, StepResult } from './runner'
+
+export async function determineShouldBufferStep(
```

Comment:
Also I just realized the naming is also problematic because `determine` implies to me that this has no side effects - it just makes a decision (and returns a boolean or something). Instead it actually sends an event to the buffer.

More like `sendEventToBufferIfNeeded` or something